packages/subgraph/src/mappings/superfluidGovernance.ts

```diff
@@ -41,6 +46,29 @@ export function handleRewardAddressChanged(event: RewardAddressChanged): void {
     ev.rewardAddress = event.params.rewardAddress;
     ev.save();
 
+    // Update reward address for token or default governance configs
+    if (event.params.superToken.equals(ZERO_ADDRESS)) {
+        let defaultGovernanceConfigs = DefaultGovernanceConfigs.load(
+            ZERO_ADDRESS.toHexString()
+        );
+        if (defaultGovernanceConfigs) {
+            defaultGovernanceConfigs.rewardAddress = event.params.rewardAddress;
```

Comment:
I don't think `updated*` fields are getting updated on this code path.

Reply:
I am assuming you are referring to governance configs being updated here and that case is handled here.

Reply:
I mean I don't notice these getting updated:
<img width="789" alt="image" src="https://user-images.githubusercontent.com/10894666/223146550-2884ed8a-660f-4e78-bd6e-c468c80fb6da.png">